src/components/KeyboardDismissableFlatList/KeyboardDismissableFlatListContext.tsx

```diff
@@ -0,0 +1,127 @@
+import PropTypes from 'prop-types';
```

Comment:
Do we really need to have the keyboard dismissal logic in a context? I think this might cause some unnecessary re-renders. Can't we have just move this logic to a hook and then use it in `KeyboardDismissableFlatList`? This is not used across multiple components, right?